libs/components/forms/src/lib/modules/radio/radio.component.ts

```diff
@@ -201,11 +216,13 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
    */
   @Input()
   public get radioType(): string {
-    return this._radioType || 'info';
+    return this.#_radioType;
   }
-  public set radioType(value: string) {
+  public set radioType(value: string | undefined) {
     if (value) {
-      this._radioType = value.toLowerCase();
+      this.#_radioType = value.toLowerCase();
```

Comment:
Probably should use `toLocaleLowerCase()`?

Reply:
Fixed